### PR TITLE
v2022.2.1.0 build

### PR DESCRIPTION
**Pull request by psteyer**

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
need to add the `dev_url` the following looks like a good candidate:
https://github.com/python/typeshed

need to add the `doc_url` the following looks like a good candidate:
https://typing.readthedocs.io/en/latest/

**Comment by skupr-anaconda** [comment]
Could you update the version to **2022.4.0.0**? See https://github.com/conda-forge/types-pytz-feedstock/blob/bea62fd883adbc47675006f8fd4b2d2a56515a45/recipe/meta.yaml#L2

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,22 +9,23 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/types-pytz-{{ version }}.tar.gz
   sha256: 47cfb19c52b9f75896440541db392fd312a35b279c6307a531db71152ea63e2b
+  skip: True # [py<36]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:12]: The skip should be `py<37`, see https://github.com/python/typeshed#about.
And it must be not in `source` but in `build`.

**Comment by psteyer** [comment]
Closing this PR and building `2022.4.0.0`.

New PR: https://github.com/AnacondaRecipes/types-pytz-feedstock/pull/3

**Pull request closed without merging**